pkg/resources/apiserver/deployment.go

```diff
@@ -394,6 +426,19 @@ func getApiserverFlags(data *resources.TemplateData, etcdEndpoints []string, ena
 		)
 	}
 
+	if data.IsKonnectivityEnabled() {
+		flags = append(flags,
+			"--egress-selector-config-file",
+			"/etc/kubernetes/konnectivity/egress-selector-configuration.yaml",
+			"--service-account-issuer",
```

Comment:
you cannot just append `--service-account-issuer`, `--service-account-signing-key-file` and `--api-audiences` flags like this, if they were already appended by the existing code 50 lines above

Reply:
Done.